examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -75,4 +82,6 @@ function unsubscribe()
     });    
 }
 
+var opts = cmdline.parse({host: 'http-api-host', port: 3000});
```

Comment:
If we're making `host` and `port` mandatory then I agree there's no point in having defaults, so far they were not mandatory, hence the defaults. Personally I would prefer them to be mandatory, because the fact is you have to know your hostz and portz before you run!